src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -79,11 +93,26 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
             model.UserName = model.UserName?.Trim();
             model.Email = model.Email?.Trim();
 
+            var usersOfAdminRole = await _userManager.GetUsersInRoleAsync("Administrator");
             if (!model.IsEnabled && user.IsEnabled)
             {
-                user.IsEnabled = model.IsEnabled;
-                var userContext = new UserContext(user);
-                await Handlers.InvokeAsync((handler, context) => handler.DisabledAsync(userContext), userContext, _logger);
+                if (user.UserName != httpContext.User.Identity.Name)
+                {
+                    if(usersOfAdminRole.Count == 1 && usersOfAdminRole.First().UserName == user.UserName)
```

Comment:
Can we move this condition at the beginning, because if there's one use we are 100% sure it's a super admin

Reply:
Which line you want me to move it to? Because here I want to display the proper notifier warning.

Reply:
I mean line 101, because when the admin users is 1, a notification should popup

Reply:
Ok reversed both `if` conditions priority.